tests/TestRunner/app/Inheritance/TypeScriptTests.js

```diff
@@ -238,6 +270,78 @@ describe(module.id, function () {
         expect(TNSGetOutput()).toBe('voidSelector called' +
             'variadicSelector:native x:9 called');
     });
+    it("MethodOverrides: errors", function () {
+        expect(function () {
+            var TSObjectErr1 = /** @class */ (function (_super) {
```

Comment:
You could add the TS code as a comment to see the original code. This is done in the Android Runtime tests and increases the readability.